apps/docs/docs/toolbox/accessibility_indicators/oev_gueteklassen.md

```diff
@@ -60,11 +60,18 @@ In case you need to perform analysis beyond this geofence, feel free to contact
   <div class="content">Select the <code>Reference Layer</code> for which you like to calculate the indicator. This can be any polygon feature layer.</div>
 </div>
 
-![Reference Area Selection](/img/toolbox/accessibility_indicators/gueteklassen/reference_area.png "Reference Area Selection")
-
+### Configuration
 
 <div class="step">
   <div class="step-number">5</div>
+  <div class="content">Select the <code>Catchment area type</code> for the calculation, which can be based on either <code>Buffer</code> or <code>Network</code>. </div>
```

Comment:
Thanks Noemi, this looks good to me. Could we just add a short explanation on the difference between a buffer and network based catchment area type?

"Buffers represent areas accessible from public transport stations based on distances "as-the-crow-flies", while network catchment areas consider the same distance along real-world streets and paths".

Feel free to rephrase this if you'd like.